opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/DefaultPluginRegistry.java

```diff
@@ -155,8 +161,7 @@ private <T> T beanWithNameOfType(String beanName, Class<T> pluginClass) {
                 // if not, the policy definition seems to be broken
                 error(null,"Policy class not found or not a policy class: '{}' of type {}", beanName, pluginClass);
             } else {
-                // log only for debug
-                debug("Failed to find bean {} with name {} of type {}", bean, beanName, pluginClass);
+                trace("Bean {} with name {} does not match requested type {}", bean, beanName, pluginClass);
```

Comment:
Should this trace message say something about the bean being found for the base class?

Reply:
Like `"Bean {} with name {} is a policy, but does not match requested type {}"` or something?

Reply:
That sounds good to me.